app/src/components/integrations/elements/integrationSettings/connectionSection/connectionSection.jsx

```diff
@@ -131,46 +172,96 @@ export class ConnectionSection extends Component {
-            <span className={cx('status-icon')}>
-              {Parser(isConnectionFailed ? InfoIcon : CheckIcon)}
-            </span>
-            {formatMessage(
-              isConnectionFailed ? messages.connectionFailedMessage : messages.connectedMessage,
-            )}
-          </span>
-          {isConnectionFailed && (
-            <span className={cx('failed-connection-message')}>{failedConnectionMessage}</span>
-          )}
-        </div>
-        {editAuthConfig && (
-          <Fragment>
-            <div className={cx('connection-status-block', 'auth-info-block')}>
-              {editAuthConfig.content}
+      <>
+        {!connected && (
+          <div className={cx({ 'with-global-message': blocked })}>
+            <SystemMessage
+              header={formatMessage(messages.connectionFailedHeader)}
+              mode="error"
+              caption={formatMessage(messages.connectionFailedCapture)}
+            >
+              {formatMessage(messages.connectionFailedDescription, {
+                pluginName: PLUGIN_NAME_TITLES[pluginName] || pluginName,
+              })}
+            </SystemMessage>
+          </div>
+        )}
+        {availableProjectIntegrations.length > 0 && blocked && (
+          <SystemMessage header={formatMessage(messages.warningMessage)} mode="warning">
+            {formatMessage(messages.warningMessageDescription)}
+          </SystemMessage>
+        )}
+
+        <div
+          className={cx('connection-section', {
+            'connection-section-with-message':
+              !connected || (availableProjectIntegrations.length > 0 && blocked),
+          })}
+        >
+          <div className={cx('integration-info-block')}>
+            <div className={cx('sub-header-block')}>
+              <div className={cx('general-info')}>
+                <h1>{name}</h1>
+                <p>
+                  {creator} on {moment(creationDate).format('ll')}
+                </p>
+              </div>
+              <div
+                className={cx('connection-block', {
+                  'connection-block-failed': !connected,
+                })}
+              >
+                {Parser(connected ? Tick : ErrorIcon)}
+                <p>
+                  {formatMessage(
+                    connected ? messages.connectedMessage : messages.connectionFailedMessage,
+                  )}
+                </p>
+              </div>
             </div>
+            {integrationParameters.url && (
+              <div className={cx('sub-block')}>
+                <span>{formatMessage(messages.linkTo)}</span>
+                <p>{integrationParameters.url}</p>
+              </div>
+            )}
+            {integrationParameters.project && (
+              <div className={cx('sub-block')}>
+                <span>{formatMessage(messages.projectKeyIn)}</span>
+                <p>{integrationParameters.project}</p>
+              </div>
+            )}
```

Comment:
Seems like you hardcoded these fields for BTS integrations.
The integration may contain any auth related fields with any names.
So I suggest to leave this responsibility for a particular integration type.